### PR TITLE
#1720 - changing xunit 2.1.0 references to 2.0.0

### PR DESCRIPTION
**Pull request by voltcode**
unifying all 2.0.0 and 2.1.0 xunit references to 2.0.0 strict to get MNTK cluster tests working

### CODE REVIEW

**Comment by voltcode** [comment]
I'm not sure if the error message is related to my xunit change. Is this something related to F#, my changes shouldn't affect  F# parts, because I have only touched c# projects.

File: src/contrib/cluster/Akka.Cluster.Sharding.Tests/Akka.Cluster.Sharding.Tests.csproj [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\packages\xunit.core.2.0.0\build\portable-net45+win+wpa81+wp80+monotouch+monoandroid+Xamarin.iOS\xunit.core.props" Condition="Exists('..\..\..\packages\xunit.core.2.0.0\build\portable-net45+win+wpa81+wp80+monotouch+monoandroid+Xamarin.iOS\xunit.core.props')" />
```
Comment (Danthar) [inline src/contrib/cluster/Akka.Cluster.Sharding.Tests/Akka.Cluster.Sharding.Tests.csproj]: Why is there a project ref for this? This PR should only change version numbers for which version of xunit is linked. This addition seems unneeded.

File: src/contrib/cluster/Akka.Cluster.Sharding.Tests/Akka.Cluster.Sharding.Tests.csproj [hunk 2]
```diff
@@ -122,12 +125,20 @@
...
-    <None Include="packages.config" />
+    <None Include="packages.config">
+      <SubType>Designer</SubType>
+    </None>
   </ItemGroup>
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
```
Comment (Danthar) [inline src/contrib/cluster/Akka.Cluster.Sharding.Tests/Akka.Cluster.Sharding.Tests.csproj]: please get rid of nuget package restore stuff. We handle that via our build script.

**Comment by voltcode** [comment]
commited changes according to the remarks

**Comment by voltcode** [comment]
@Danthar I addressed your remarks, would you mind taking another look?

**Comment by Danthar** [comment]
Looks alot better now. Thx. Could you please squash your commits. We endeavour to keep our merges clean in order to keep our sanity when having to compare changes :+1: 

The MNTK-error is a race condition unrelated to your PR. 
The windows-unit-tests one. Not sure yet. We are investigating because we are seeing it in other places as well.

**Comment by voltcode** [comment]
squashed the commits successfully @Danthar

**Comment by Danthar** [comment]
:+1:

**Pull request merged**